docs/flow/reference/components/remote.s3.md

```diff
@@ -73,4 +78,7 @@ the watched file succeeded.
 
 `remote.s3` does not expose any component-specific debug metrics.
 
-[secret]: ../secrets.md#is_secret-argument-in-components
+## Other S3 compatible systems
+
+Other S3 compatible systems can be read with `remote.s3` but may require specific authentication environment variables. There is no guarantee that `remote.s3` will work with non-AWS S3 systems. 
```

Comment:
I think this should be closer to the top since it might be something people want to know immediately.

WDYT about putting it in a blockquote after the description of the component?

```markdown
# remote.s3

`remote.s3` exposes the string contents of a 
file located in [AWS S3](https://aws.amazon.com/s3/)  
to other components. The file will be polled for 
changes so that the most recent content is always 
available.

> **NOTE**: Other S3 compatible systems can be read 
> with `remote.s3` but may require specific 
> authentication environment variables. There is no 
> guarantee that `remote.s3` will work with non-AWS S3 
> systems. 
```

Reply:
Sounds reasonable.